### PR TITLE
Fix #12281 Copying a user resets SSL to None

### LINKED ISSUE #12281: Copying a user resets SSL to None
### Steps to reproduce
1. Create a user with the Require SSL radio button selected
2. Copy that user

### Expected behaviour
The new user should have Require SSL set

### Actual behaviour
The new user defaults to "None"

### Server configuration
**Database:**
MySQL  5.5.49-0+deb8u1 - (Debian) 
**phpMyAdmin version:**
git master
### Client configuration
**Browser:**
Firefox
**Operating system:**
MacOS

### PR DESCRIPTION
**Pull request by madhuracj**
Before submitting pull request, please check that every commit:
- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them

Signed-off-by: Madhura Jayaratne madhura.cj@gmail.com

### CODE REVIEW

**Comment by ibennetch** [comment]
Works great for me. Nice job.

**Comment by ibennetch** [comment]
If I add a user to a usergroup then copy that user, it probably should keep the usergroup as well. Is that reasonable to fix in this patch or should I open a new issue?

File: ChangeLog [hunk 1]
```diff
@@ -6,6 +6,7 @@ phpMyAdmin - ChangeLog
 - issue #12279 Fixed error reporting on connect problems
 - issue #12290 Fixed export of tables without explicitly set engine
 - issue #12285 Designer JavaScript error: Show/Hide tables list
+- issue #12281 Copying a user resets SSL to None
```
Comment (ibennetch) [inline ChangeLog]: Given the expanded scope of this patch, I think I would change this to "Keep user attributes (privileges, authentication mode, etc) when copying a user"

**Comment by madhuracj** [comment]
Given that user group is a phpMyAdmin feature rather than a MySQL privilege/feature, I am not sure whether this should be set. If you think it should be set for the new user, please open a separate ticket for it.

**Pull request merged**